mypy error when using `from simple_pid import PID`

When I use `from simple_pid import PID` and check the code with `mypy --strict` I get the following error:
```
$ mypy --strict test.py
test.py:1: error: Module "simple_pid" does not explicitly export attribute "PID"; implicit reexport disabled
```
I guess this is an issue with the `from .PID import PID` in `__init__.py`? If I change it to `from .PID import PID as PID` it works.

Also I think it is strange that the import in `__init__.py` shadows the `PID` module in `simple_pid`.

Maybe it would be better to rename the module to `pid`?

--- a/simple_pid/__init__.py
+++ b/simple_pid/__init__.py
@@ -1,1 +1,1 @@
-from .PID import PID
+from simple_pid.PID import PID as PID

